src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,4 +1,49 @@
-const convertInclude = ({ href, url, type, ...rest }) => {
+import 'isomorphic-fetch';
+import nodeLogger from '#lib/logger.node';
+
+const logger = nodeLogger(__filename);
+
+// this defaults all VJ and IDT2 includes to the specified urls for now
+// should be removed once mozart routes have been created
```

Comment:
Can we add the PR number which resolves this TODO? https://github.com/bbc/simorgh-infrastructure/issues/860